stacks/web-application.yml

```diff
@@ -0,0 +1,257 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Web application with optional worker running in Docker
+Conditions:
+  CreateWorkerResources: !Equals [!Ref CreateWorker, true]
+Parameters:
+  # VPC ########################################################################
+  VPC:
+    Type: "AWS::EC2::VPC::Id"
+  # Load Balancer ##############################################################
+  PlatformALBDNSName:
+    Type: String
+  PlatformALBFullName:
+    Type: String
+  PlatformALBCanonicalHostedZoneID:
+    Type: String
+  PlatformALBHTTPListenerArn:
+    Type: String
+  PlatformALBHTTPSListenerArn:
+    Type: String
+  PlatformALBListenerPriorityPrefix:
+    Type: String
+  # ECS Cluster ################################################################
+  ECSCluster:
+    Type: String
+  ECSServiceAutoscaleIAMRoleArn:
+    Type: String
+  ECSServiceIAMRole:
+    Type: String
+  # Misc #######################################################################
+  OpsDebugMessagesSnsTopicArn:
+    Type: String
+  OpsErrorMessagesSnsTopicArn:
+    Type: String
+  EnvironmentType:
+    Type: String
+  EnvironmentTypeAbbreviation:
+    Type: String
+  EcrRegion:
+    Type: String
+  SecretsBase:
+    Type: String
+  # Shared ENV #################################################################
+  # App ECS #################################################################
+  EcrImageTag:
+    Type: String
+  # App ENV #################################################################
+  AppName:
+    Type: String # castle, crier, cms, feeder
+  SecretsVersion:
+    Type: String
+  CreateWorker:
+    Type:  String
+  ContainerPort:
+    Type: String
+    Default: 3000
+  ContainerMemory:
```

Comment:
currently yes, the examples I looked at have these as the same already. Generally the amount of memory and CPU will be similar since it is running the same app and docker image. We can break these out if needed later.